forge-gui/res/cardsfolder/upcoming/the_sea_devils.txt

```diff
@@ -1,11 +1,11 @@
-Name:The Sea Devils
-ManaCost:2 G
-Types:Enchantment Saga
-K:Saga:3:DBToken,DBToken,DBEffect
-SVar:DBToken:DB$ Token | TokenScript$ g_2_2_alien_salamander_islandwalk | SpellDescription$ Create a 2/2 green Alien Salamander creature token with islandwalk. (It can't be blocked as long as defending player controls an Island.)
-SVar:DBEffect:DB$ Effect | Triggers$ TrigDamage | SpellDescription$ Until end of turn, whenever a Salamander deals combat damage to a player, it deals that much damage to target creature that player controls.
-SVar:TrigDamage:Mode$ DamageDone | ValidSource$ Salamander | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDealDamage | TriggerDescription$ Whenever a Salamander deals combat damage to a player, it deals that much damage to target creature that player controls.
-SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Creature.ControlledBy TriggeredTarget | DamageSource$ TriggeredSource | NumDmg$ X | TgtPrompt$ Select target creature that player controls
-SVar:X:TriggerCount$DamageAmount
-DeckHas:Ability$Token & Type$Salamander|Alien
-Oracle:(As this Saga enters and after your draw step, add a lore counter. Sacrifice after III.)\nI, II — Create a 2/2 green Alien Salamander creature token with islandwalk. (It can't be blocked as long as defending player controls an Island.)\nIII —Until end of turn, whenever a Salamander deals combat damage to a player, it deals that much damage to target creature that player controls.
+Name:The Sea Devils
+ManaCost:2 G
+Types:Enchantment Saga
+K:Saga:3:DBToken,DBToken,DBEffect
+SVar:DBToken:DB$ Token | TokenScript$ g_2_2_alien_salamander_islandwalk | SpellDescription$ I, II — Create a 2/2 green Alien Salamander creature token with islandwalk. (It can't be blocked as long as defending player controls an Island.)

```

Comment:
the chapters should already be added by auto builder anyway
best to check such stuff ingame